Havit/Havit/Global/UIComponent/ContentsCollectionViewCell.swift

```diff
@@ -0,0 +1,167 @@
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        contentView.addSubview(mainImgView)
+        
+        titleLabel = {
+            let label = UILabel()
+            label.text = "슈슈슉 이것은 제목입니다 슈슉 슉슉 이것"
+            label.numberOfLines = 0
+            
+            return label
+        }()
+        contentView.addSubview(titleLabel)
+        
+        subtitleLabel = {
+            let label = UILabel()
+            label.text = "슈슈슉 이것은 제목입니다 슈슉 슉"
+            
+            return label
+        }()
+        contentView.addSubview(subtitleLabel)
+        
+        dateLabel = {
+            let label = UILabel()
+            label.text = "2021. 11. 24"
+            
+            return label
+        }()
+        contentView.addSubview(dateLabel)
+        
+        linkLabel = {
+            let label = UILabel()
+            label.text = "www.beansbin.oopy.io"
+            
+            return label
+        }()
+        contentView.addSubview(linkLabel)
+        
+        
+        alarmLabel = {
+            let label = UILabel()
+            label.text = "2021. 11. 17 오전 12:30 알림 예정"
+            
+            return label
+        }()
+        contentView.addSubview(alarmLabel)
+        
+        alarmImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        mainImgView.addSubview(alarmImgView)
+        
+        moreButton = {
+            let button = UIButton()
+            button.setTitle("...", for: .normal)
+            return button
+        }()
+        contentView.addSubview(moreButton)
+        
+        isReadImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        contentView.addSubview(isReadImgView)
+        
+    }
+    
+    func setAutoLayouts() {
+        mainImgView.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(contentView).offset(12)
+            make.leading.equalTo(contentView).offset(16)
+            make.trailing.equalTo(contentView).offset(19)
+            make.height.equalTo(108)
+        }
+        
+        titleLabel.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(mainImgView).offset(12)
+            make.top.equalTo(contentView).offset(12)
+            make.trailing.equalTo(moreButton).offset(17)
+        }
+        
+        subtitleLabel.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(mainImgView).offset(12)
+            make.top.equalTo(titleLabel).offset(6)
+        }
+        
+        dateLabel.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(mainImgView).offset(12)
+            make.top.equalTo(subtitleLabel).offset(15)
+        }
+        linkLabel.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(dateLabel).offset(0)
```

Comment:
P3;
굳이 정해야할 offset이 없다면 안써주셔도 됩니다.
```suggestion
            make.leading.equalTo(dateLabel)
```

Reply:
넵 대장님! 😃😃